### PR TITLE
Adds instant-orderable insulated gloves to the black market.

### PR DESCRIPTION
**Pull request by MadmanMartian**
Available at roughly the same price as the full supply pack from cargo

:cl:
 * rscadd: Adds yellow gloves to the black market.

### CODE REVIEW

**Comment by boy2mantwicethefam** [comment]
It's tactical!

**Comment by jknpj** [comment]
I get the reference

**Comment by ShiftyRail** [comment]
Sneaky coders

File: code/modules/clothing/gloves/color.dm [hunk 1]
```diff
@@ -60,6 +60,14 @@
 		return 1
 	return
 
+/obj/item/clothing/gloves/finyellow
+	name = "insulated gloves"
+	desc = "These gloves will protect the wearer from ecstatic shock."
+	icon_state = "yellowF"
+	item_state = "yellow"
+	siemens_coefficient = 1
```
Comment (SonixApache) [inline code/modules/clothing/gloves/color.dm:68]: 😒

**Comment by SonixApache** [comment]
i don't get the reference

**Comment by ghost** [comment]
What if instead of always being fake, you made it have a 15% fake, 70% real, 15% to be fake and have superglue in them

File: code/modules/clothing/gloves/color.dm [hunk 2]
```diff
@@ -60,6 +60,14 @@
 		return 1
 	return
 
+/obj/item/clothing/gloves/finyellow
+	name = "insulated gloves"
+	desc = "These gloves will protect the wearer from ecstatic shock."
```
Comment (brndd) [inline code/modules/clothing/gloves/color.dm:65]: change this to "eclectic shock" please

**Review by brndd — CHANGES_REQUESTED** [review]

**Comment by Intigracy** [comment]
This is just a trap for people who don't read github.

At least the fake insulateds that spawn on the map have a chance of being real.

**Comment by Kurfursten** [comment]
Needs accurate changelog. No lying in cls except for april fools, even if the desc gives it away.

**Comment by MadmanMartian** [comment]
Giving away everything in the changelog is pretty boring

**Comment by Intigracy** [comment]
noob traps are still gay

**Comment by MadmanMartian** [comment]
Budget insulated gloves

**Comment by Intigracy** [comment]
Budget insulated gloves have a random coefficient assigned when they're created.

These are always fake.

**Comment by Kurfursten** [comment]
Let's compromise on "yellow" gloves. Fix conflicts and you're in.

**Comment by MadmanMartian** [comment]
fun is verboten

**Pull request closed without merging**